src/modules/dashboard/application/use-cases/widgets/__tests__/export-widget-data.use-case.spec.ts

```
@@ -0,0 +1,211 @@
+    });
+
+    it('should export data in JSON format', async () => {
+      const query: ExportQueryDto = {
+        format: 'json',
+      };
+
+      const result = await useCase.execute(widgetId, query);
+
+      expect(result.contentType).toBe('application/json');
+      expect(result.filename).toBe('widget-widget-123-export-2024-01-01.json');
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)